source/includes/cross-version-sync.rst

```diff
@@ -1,9 +1,3 @@
 Starting in 1.7.0, ``mongosync`` can perform version migration from 
 certain lower major version source clusters to certain higher major
 version destination clusters. 
-
-Cross-version migration requires additional preparation and
-configuration when migrating from a pre-6.0 release. To perform a cross-version 
-migration from a pre-6.0 version of the MongoDB Server using
-``mongosync``, please `contact <https://mongodb.com/contact>`__ your
-account team to inquire about Professional Services.
```

Comment:
To double check, the other blurb on my mind is this one: https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/supported-server-version/#synchronize-data-between-clusters-running-older-mongodb-server-versions
"Starting in version 1.2, mongosync can sync between clusters running some older (pre-6.0) releases of MongoDB Server. This feature requires additional preparation and configuration in pre-6.0 releases.

If you would like to migrate pre-6.0 versions of MongoDB Server using mongosync, please [contact](https://mongodb.com/contact) your account team to inquire about Professional Services."

I didn't see this blurb in the staging link but I still see the text here - just want to confirm that it's been removed. Thanks!

Reply:
Updated.